src/mysql.erl

```diff
@@ -25,6 +25,9 @@
 %% gen_server is locally registered.
 -module(mysql).
 
+-include("records.hrl").
+-include("server_status.hrl").
+
 -export([start_link/1, query/2, query/3, query/4, execute/3, execute/4,
```

Comment:
I'd like the public interface, i.e. the exports, to be at the top of the file. Includes are for the implementation and need not be the first thing visible in the file. See also the comment where these are moved from.